app/uk/gov/hmrc/agentsexternalstubs/controllers/EnrolmentStoreProxyStubController.scala

```diff
@@ -53,8 +53,17 @@ class EnrolmentStoreProxyStubController @Inject() (
         delegated <- if (`type` == "all" || `type` == "delegated") {
                        enrolmentKey.service match {
                          case "IR-PAYE" | "IR-SA" =>
-                           usersService.findUserIdsByDelegatedEnrolmentKey(enrolmentKey, session.planetId)(1000)
-                         case _ => usersService.findUserIdsByDelegatedEnrolmentKey(enrolmentKey, session.planetId)(1000)
+                           usersService.findUserIdsByAssignedDelegatedEnrolmentKey(
```

Comment:
we don't need the 2 cases now as the code looks the same for both?

Reply:
True! Well spotted